Library/Homebrew/dev-cmd/boneyard-formula-pr.rb

```diff
@@ -0,0 +1,104 @@
+# Creates a pull request to boneyard a formula.
+#
+# Usage: brew boneyard-formula-pr [options...] <formula-name>
+#
+# Options:
+#   --dry-run:  Print what would be done rather than doing it.
+
+require "formula"
+require "utils/json"
+require "fileutils"
+
+module Homebrew
+  def boneyard_formula_pr
+    formula = ARGV.formulae.first
+    odie "No formula found!" unless formula
+
+    formula_path = formula.path
+    core = formula_path.to_s.include? "homebrew-core"
+    if ARGV.dry_run?
+      ohai "brew update"
+      ohai "brew tap homebrew/boneyard"
+      ohai "cp #{formula_path} #{HOMEBREW_LIBRARY}/Taps/homebrew/homebrew-boneyard"
```

Comment:
`#{HOMEBREW_LIBRARY}/Taps/homebrew/homebrew-boneyard` is used several times. Might make sense to put it into a constant (`BONEYARD_PATH`?) at the beginning of the command to make the rest of the code more readable.